zebra/zebra_ptm.c

```diff
@@ -908,21 +896,18 @@ zebra_ptm_bfd_dst_deregister (struct zserv *client, int sock, u_short length,
           ptm_lib_append_msg(ptm_hdl, out_ctxt,
                               ZEBRA_PTM_BFD_SRC_IP_FIELD, buf);
         }
-#ifdef HAVE_IPV6
       else
         {
           inet_ntop(AF_INET6, &src_p.u.prefix6, buf, sizeof(buf));
           ptm_lib_append_msg(ptm_hdl, out_ctxt,
                               ZEBRA_PTM_BFD_SRC_IP_FIELD, buf);
         }
-#endif /* HAVE_IPV6 */
       if (zvrf_id (zvrf) != VRF_DEFAULT)
 	ptm_lib_append_msg(ptm_hdl, out_ctxt, ZEBRA_PTM_BFD_VRF_NAME_FIELD,
 			   zvrf_name (zvrf));
     }
```

Comment:
Please, simplify the if / else statement by putting ptm_lib_append_msg() function call outside the if / else like you propose in the previous modification (line 888 -> 874,876)